WFLY-6459_EAP7-838_JAX-RS_2.1.adoc

```diff
@@ -0,0 +1,60 @@
+= WFLY-6459 / EAP7-838 JAX-RS 2.1 Support
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+This issue has two parts.
+Firstly it brings JAX-RS 2.1 into Wildfly.
+Secondly, it introduces support for switch between JAX-RS 2.0 (EE7) and JAX-RS 2.1 (EE 8) for the sake of certification.
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-838[EAP7-838]
+* {issue-base-url}/WFLY-6459[WFLY-6459]
+
+=== Related Issues:
+
+* {issue-base-url}/WFLY-6460[WFLY-6460]
+* {issue-base-url}/WFLY-7185[WFLY-7185]
+* {issue-base-url}/RESTEASY-1679[RESTEASY-1679]
+
```

Comment:
I think tracker for backward compatibility should be linked too

  https://issues.jboss.org/browse/RESTEASY-1794 / https://github.com/resteasy/Resteasy/pull/1397

Reply:
@asoldano please fix, or discuss :)

Reply:
@kabir @rsvoboda done